docs/data-sources/cnad_advanced_instances.md

```diff
@@ -0,0 +1,72 @@
+---
+subcategory: "Cloud Native Anti-DDoS Advanced"
+---
+
+# huaweicloud_cnad_advanced_instances
+
+Use this data source to get the list of CNAD advanced instance.
+
+## Example Usage
+
+```hcl
+variable "package_name" {}
+
+data "huaweicloud_cnad_advanced_instances" "test" {
+  package_name = var.package_name
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `package_id` - (Optional, String) Specifies the package id.
+
+* `package_name` - (Optional, String) Specifies the package name.
+
+* `instance_type` - (Optional, String) Specifies the package instance type. Valid values are:
+  + **cnad_pro**: Professional Edition.
+  + **cnad_ip**: Standard Edition.
+  + **cnad_ep**: Platinum Edition.
+  + **cnad_full_high**: Unlimited Protection Advanced Edition.
+  + **cnad_vic**: On demand Version.
+  + **cnad_intl_ep**: International Station Platinum Edition.
+
+* `region` - (Optional, String) Specifies the region in which to query the data source.
+  + For Unlimited Protection Advanced Edition, support regions are: **cn-north-2**, **cn-north-4**, **cn-east-3**
+    and **cn-south-1**.
+  + For Unlimited Protection Basic Edition, support regions are: **cn-north-4**, **cn-east-3**, **cn-south-1**
+    and **cn-southwest-2**.
+  + For Cloud Native Anti-DDoS Standard, support regions are: **cn-north-4**, **cn-east-3** and **cn-south-1**.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The resource ID.
+
+* `packages` - Indicates the list of the advanced instances
+  The [packages](#CNADAdvancedInstances_Packages) structure is documented below.
+
+<a name="CNADAdvancedInstances_Packages"></a>
+The `packages` block supports:
+
+* `package_id` - Indicates the package id.
+
+* `package_name` - Indicates the package name.
+
+* `region` - Indicates the region where the package belongs to.
+
+* `instance_type` - Indicates the instance type of the package.
+
+* `protection_type` - Indicates the protection type of the package.
+
+* `ip_num` - Indicates ip num of the package.
```

Comment:
```suggestion
* `ip_num` - Indicates the ip num of the package.
```
all others description contians the word of the, it is better to maintain consistent format

Reply:
OK